### DISCUSSION TITLE
How to navigate folders?

### DISCUSSION THREAD

**Discussion by galacticstudios** [keirf/flashfloppy#846] [Q&A]
This is perhaps a newbie question, but how do I navigate folders?

In ff.cfg, I've set 
`autoselect-folder-secs = 0`
and
`twobutton-action = rotary`

But when I turn the knob to a folder name and press B1 or B2, it just goes to the previous or next file or folder name. Pressing the rotary button does nothing.

What I'd like is to be able to turn the knob, then press the knob in.

How can I configure that? (I have FlashFloppy 3.41.)

Thank you.

**Comment by keirf** [comment]
Pushing the rotary should act as select. If it does nothing the rotary may not be hooked up correctly. For example it should enable main menu when pressed with no usb stick inserted, too.

**Comment by galacticstudios** [comment]
Well, I opened up the Gotek, glared at it, and it started working. So I don't know what was wrong before, but it's OK now.

Thank you.
